bot.py

```diff
@@ -96,44 +94,50 @@ async def get_rank(ctx, name: str):
     try:
         summoner_info = get_summoner_rank(name)
 
-        embed = discord.Embed(title="Solo/Duo Rank", color=discord.Color.dark_gray())
-
-        summoner_name = summoner_info["user_name"]
-
-        # Removing space of the summoner name to access op.gg url of the summoner
-        summoner_name_opgg = summoner_name.replace(" ", "")
-
-        # Add author, thumbnail, fields, and footer to the embed
-        embed.set_author(
-            name=summoner_name,
-            url=f"https://na.op.gg/summoner/userName={summoner_name_opgg}",
-            icon_url=summoner_info["summoner_icon_image_url"],
+        embed_data = EmbedData()
+        embed_data.title = "Solo/Duo Rank"
+        embed_data.description = (
+            f"`All Data from NA server`\n\n <@!{bot.user.id}> <command>"
         )
+        embed_data.color = discord.Color.dark_gray()
 
-        # Get image of tier by path
-        tier_image = summoner_info["tier_image"]
-        file = discord.File(tier_image)
-        # Need to get the filename in order to attach to the thumbnail
+        # Add author, thumbnail, fields, and footer to the embed
+        embed_data.author = {}
+        embed_data.author = {
+            "name": summoner_info["user_name"],
+            # For op.gg link, we have to remove all whitespace.
+            "url": "https://na.op.gg/summoner/userName={0}".format(
+                summoner_info["user_name"].replace(" ", "")
+            ),
```

Comment:
@jjaykim @jung1411 Take a look at this string formatting;
```
"https://na.op.gg/summoner/userName={0}".format(summoner_info["user_name"].replace(" ", ""))
```
 This simplifies the code from declaring unnecessary variables

Reply:
This method was easy to find. you guys should've found this and implement it this way imo.